cloudflared/build.yaml

```diff
@@ -1,7 +1,10 @@
 ---
 build_from:
-  aarch64: ghcr.io/hassio-addons/base/aarch64:11.0.1
-  amd64: ghcr.io/hassio-addons/base/amd64:11.0.1
-  armhf: ghcr.io/hassio-addons/base/armhf:11.0.1
-  armv7: ghcr.io/hassio-addons/base/armv7:11.0.1
-  i386: ghcr.io/hassio-addons/base/i386:11.0.1
+  aarch64: ghcr.io/hassio-addons/base/aarch64:11.1.0
+  amd64: ghcr.io/hassio-addons/base/amd64:11.1.0
+  armhf: ghcr.io/hassio-addons/base/armhf:11.1.0
+  armv7: ghcr.io/hassio-addons/base/armv7:11.1.0
+  i386: ghcr.io/hassio-addons/base/i386:11.1.0
+codenotary:
+  base_image: codenotary@frenck.dev
```

Comment:
Should we use the base_image from frenck one or "notary@home-assistant.io" as suggested in the [official documentation](https://developers.home-assistant.io/docs/add-ons/configuration/#:~:text=codenotary.base_image)?

Reply:
At the moment we rely on the hassio-addons base image. The log level logic I implemented depends on parts of this base image. 

Also this part of the init stage comes from [hassio-addons](https://github.com/hassio-addons/addon-base/tree/main/base/rootfs/etc/cont-init.d) and is missing in the official images: 
```bash
[cont-init.d] 00-banner.sh: executing... 
-----------------------------------------------------------
 Add-on: Cloudflared
 Use a Cloudflared tunnel (formerly Argo Tunnel) to remotely connect to Home Assistant without opening any ports
-----------------------------------------------------------
 Add-on version: 0.6.0
 You are running the latest version of this add-on.
 System: Home Assistant OS 7.5  (amd64 / qemux86-64)
 Home Assistant Core: 2022.3.4
 Home Assistant Supervisor: 2022.03.4
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
Log level is set to INFO                           <--- This init scripts writes an environment variable to S6 which I use for cloudflared log level determination
[cont-init.d] 01-log-level.sh: exited 0.
```

I think the decision we have to make is in terms of trust. Since I assume that we can trust both sources, I would suggest staying with the current image.